transform.go

```diff
@@ -431,6 +436,15 @@ func (t *Transformer) makeRhsParam(expr Expression) interface{} {
 	}
 }
 
+func (t *Transformer) previousNotNode() bool {
```

Comment:
This is from the old implementation of NotExpr, which is already completed

Reply:
Removed